airflow/providers/amazon/aws/hooks/lambda_function.py

```diff
@@ -93,6 +93,12 @@ def create_lambda(
         code_signing_config_arn: str | None = None,
         architectures: list[str] | None = None,
     ) -> dict:
+        if package_type == "Zip":
+            if handler is None:
+                raise ValueError("Parameter 'handler' is required if 'package_type' is 'Zip'")
+            if runtime is None:
+                raise ValueError("Parameter 'runtime' is required if 'package_type' is 'Zip'")
```

Comment:
These should be TypeError to mirror Python’s default behaviour.

```pycon
>>> def f(*, a): pass
... 
>>> f()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: f() missing 1 required keyword-only argument: 'a'
```

Reply:
Good call! Fixed